src/pattern-library/routes.ts

```diff
@@ -1,51 +1,44 @@
 import type { FunctionComponent } from 'preact';
 
-import LibraryHome from './components/LibraryHome';
+// Legacy pattern-library pages
+import ButtonComponents from './components/patterns/ButtonComponents';
+import ContainerComponents from './components/patterns/ContainerComponents';
+import DialogComponents from './components/patterns/DialogComponents';
+import FormComponents from './components/patterns/FormComponents';
+import IconComponents from './components/patterns/IconComponents';
+import LinkComponents from './components/patterns/LinkComponents';
+import PanelComponents from './components/patterns/PanelComponents';
+import SpinnerComponents from './components/patterns/SpinnerComponents';
+import TableComponents from './components/patterns/TableComponents';
+import ThumbnailComponents from './components/patterns/ThumbnailComponents';
```

Comment:
The block of legacy page imports used to be at the bottom. However, I couldn't find an easy way to keep that behavior. The closest I found was a way to keep them as their own group, but at the top instead of the bottom (see [this comment](https://github.com/hypothesis/frontend-shared/pull/877/files#r1118444666)).

Since we are soon removing this, I thought this would be good enough for now, but if it is preferred to keep the previous behavior, I can check a bit further and try to find a better regular expression.

For context, the reason it is at the top is because this plugin matches imports against regular expressions defined in `importOrder` prettier config. Each regexp defined there becomes an import group.

The one we usually use (`^[./]`) is too broad, and matches legacy pages as well, resulting in a mixture of imports.

I tried adding `^.\/components\/patterns\/(.*)Components$` after, which only matches legacy pages, but for the reason explained above, it does not work unless we but it before `^[./]`.

I have tried using a regexp which is "the opposite" of `^.\/components\/patterns\/(.*)Components$`, but it's tricky and has weird side effects.

Reply:
@lyzadanger your feedback here would be very valuable.